HW2_nekrasova/calculator.py

```diff
@@ -0,0 +1,62 @@
+def main(expression: str):
+    expression_array = expression.split(' ')
```

Comment:
```suggestion
    expression_array = expression.split()
```
Если хочется сделать сплит по пробельным символам, то советую такую конструкцию. В случае введения двойного пробела ваш вариант вставит " " в список, а предложенная - нет. 😌